Neos.Flow/Classes/Mvc/Controller/Argument.php

```diff
@@ -212,12 +218,15 @@ public function setValue($rawValue): Argument
             $this->value = $rawValue;
             return $this;
```

Comment:
Am I correct that until now a `setValue` without property mapping (because value was already an object) would skip the validation? Was this intended? Why?

Reply:
🤷‍♂️